### DISCUSSION TITLE
How to color floaterm?

### DISCUSSION THREAD

**Discussion by 9mm** [neovide/neovide#2022] [Q&A]
https://github.com/akinsho/toggleterm.nvim

I have searched around this plugin in depth and can't seem to find much, because most people seem to run nvim inside terminal where the colors match.

I'm trying to use toggleterm for lazygit and it looks abysmal... but it would also be nice to have the terminal colors in general match my theme.

Is that possible somehow? I thought this might be a good place to ask considering its specifically related to how GUI handles colors. I'm not sure how that works with a terminal interface... inside a gui

Thanks!

**Comment by fredizzimo** [comment]
You should open an issue in your color scheme repository, it should set the colors. But you can do it manually https://neovide.dev/faq.html#the-terminal-displays-fallback-colorsterminal-does-not-show-my-colors

**Comment by 9mm** [comment]
Thanks, I thought that might be the case, and catpuccin is doing that:

https://github.com/catppuccin/nvim/blob/85e93601e0f0b48aa2c6bbfae4d0e9d7a1898280/lua/catppuccin/groups/terminal.lua#L6

I was thinking maybe there was something else I had to do with the GUI settings

**Reply by fredizzimo** [reply]
That doesn't look right, so I had to check

https://github.com/catppuccin/nvim/issues/491#issuecomment-1556059724, you have to actually enable it.

**Comment by 9mm** [comment]
Uhghhhhhhhh 🤦🏻‍♂️ Ok... thank you. that was stupid
